examples/helpCommandExample.lua

```diff
@@ -0,0 +1,43 @@
+local discordia = require('discordia')
+local client = discordia.Client()
+discordia.extensions() -- load all helpful extensions
+
+local commands = {
+	ping = {
+		description = "Answers with pong.",
+		exec = function(message)
+			message.channel:send("Pong!")
+		end
+	},
+	hello = {
+		description = "Answers with world.",
+		exec = function(message)
+			message.channel:send("world!")
+		end
+	}
+}
+
+client:on('ready', function()
+	p(string.format('Logged in as %s', client.user.username))
+end)
+
+client:on("messageCreate", function(message)
+	local args = content:split(" ") -- split all arguments into a table
+
+	local command = commands[args[1]]
+	if command then -- ping or hello
+		command.exec(message) -- execute the command
+	end
+
+	if args[1] == "help" then -- display all the commands
+		local output = ""
+		for word, tbl in pairs(commands) do
+			output = output .. "Command: " .. word .. "\nDescription: " .. tbl.description .. "\n\n"
```

Comment:
Multiple concatenations and concatenations in a loop are bad. Use a buffer table and concat with newlines after the loop.